bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelCommand.java

```diff
@@ -221,12 +221,13 @@ public enum RotelCommand {
     STEREO7("7 Channel Stereo", PRIMARY_CMD, (byte) 0x5C, "7channel", "7channel"),
     STEREO9("9 Channel Stereo", "9channel", "9channel"),
     STEREO11("11 Channel Stereo", "11channel", "11channel"),
-    DSP_TOGGLE("DSP Mode Toggle", PRIMARY_CMD, (byte) 0x14),
+    PROLOGIC_TOGGLE("Dolby Pro Logic Mode Toggle", PRIMARY_CMD, (byte) 0x13),
+    DSP_TOGGLE("DSP Music Mode Toggle", PRIMARY_CMD, (byte) 0x14),
     DSP1("DSP 1", PRIMARY_CMD, (byte) 0x57),
     DSP2("DSP 2", PRIMARY_CMD, (byte) 0x58),
     DSP3("DSP 3", PRIMARY_CMD, (byte) 0x59),
     DSP4("DSP 4", PRIMARY_CMD, (byte) 0x5A),
-    PROLOGIC_TOGGLE("Dolby 3 Stereo / Pro Logic Toggle", PRIMARY_CMD, (byte) 0x53),
+    DOLBY_TOGGLE("Dolby 3 Stereo / Pro Logic Toggle", PRIMARY_CMD, (byte) 0x53),
```

Comment:
For consistency with the others?
```suggestion
    DOLBY_TOGGLE("Dolby 3 Stereo / Pro Logic Mode Toggle", PRIMARY_CMD, (byte) 0x53),
```

Reply:
I would like to keep the text present in the Rotel API documentation ... except when this description is not correct, which is the case for the command I am adding. So I don't want to add "Mode" and I chose to replace "Dolby Pro Logic Mode Toggle" by "Dolby Pro Logic Toggle".

Reply:
Just to be sure: "DSP Music Mode Toggle" should still include "Mode"?

Reply:
Yes, this is named like that in the doc.

Reply:
Please wait a second...

Reply:
Ok for final review.